src/main/java/Wolox/training/controllers/BookController.java

```diff
@@ -1,16 +1,55 @@
 package Wolox.training.controllers;
 
+import Wolox.training.exceptions.BookNotFoundException;
+import Wolox.training.models.Book;
+import Wolox.training.repositories.BookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.*;
 
 @Controller
+@RequestMapping("/api/books")
 public class BookController {
     @GetMapping("/greeting")
     public String greeting(@RequestParam(name="name", required=false, defaultValue="World") String name, Model model) {
         model.addAttribute("name", name);
         return "greeting";
     }
 
+    @Autowired
+    private BookRepository bookRepository;
+
```

Comment:
It is recommended to use **injection by constructor**, instead of **injection by field**. When you use the latter, you are coupling unit tests with the Spring context and that is not the ideal scenario.
It is worth mentioning that when there is a constructor, it is not necessary to use the annotation, Spring automatically recognizes the constructor and injects the instances.

pd: I know the trello card mentions the use of annotation, it needs to be updated.
```suggestion
    private BookRepository bookRepository;
    
    public BookController(BookRepository bookRepository) {
        this.bookRepository = bookRepository;
    }
```